.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -14,6 +14,7 @@ body:
       description: What version of our F5 NGINX Ingress Controller are you running?
       options:
         - edge
+        - 3.7.0
```

Comment:
we can skip this as its included in https://github.com/nginxinc/kubernetes-ingress/pull/6544/files#diff-16caf9c5c96b17c5ca3f5762c8f05897e148bcfeb7723f42cff8b42f089d2f65R17